docs/aws.md

```diff
@@ -74,7 +74,7 @@ aws configure # Input your credentials here
 aws iam list-users
 ```
 
-PyPi is the officially supported `aws cli` download avenue, and kops suggests using it. [More information](https://pypi.python.org/pypi/awscli) on the package. 
+PyPi is the officially supported `aws cli` download avenue, and kops suggests using it. [More information](https://pypi.python.org/pypi/awscli) on the package.
```

Comment:
Did you intend this to be in this PR?  I thought I saw the same change in a different PR